web/themes/custom/customtheme/components/01-atoms/buttons/back-to-top.js

```diff
@@ -0,0 +1,22 @@
+(function ($, Drupal, once) {
+  Drupal.behaviors.backToTop = {
+    attach: function (context, settings) {
+      var backButton = once('backToTopButton', '.font-container', context)[0];
+      $(backButton).hide();
+      $(window, context).on('scroll', function() {
+        var scrollPosition = $(window).scrollTop();
+
+        if (scrollPosition > 100) {
+          $(backButton).fadeIn();
```

Comment:
So on every scroll past 100 you are calling fadeIn(), it doesn't make sense to do it when the button is already displayed after a first call.
But that also depends on what fadeIn is doing, if it has a condition inside to do nothing if the element is already visible than it is fine.